igor-core/src/main/java/com/netflix/spinnaker/igor/build/model/GenericGitRevision.java

```diff
@@ -36,4 +36,20 @@
   private String message;
   private Instant timestamp;
   private String remoteUrl;
+
+  public GenericGitRevision withAuthorInfo(String committer, String message) {
+    this.committer = committer;
+    this.message = message;
+    return this;
+  }
+
+  public GenericGitRevision withTimestamp(Instant timestamp) {
+    this.timestamp = timestamp;
+    return this;
+  }
+
+  public GenericGitRevision withCompareUrl(String link) {
+    this.compareUrl = link;
+    return this;
+  }
```

Comment:
These two methods can be removed, they are automatically generated by the Lombok `@Wither` annotation.

Reply:
good call. didnt notice `@Wither`. removed.

Reply:
one little annoyance with `@Wither` however is that it doesnt do in-place update of the properties, so I had to capture the returned value.